src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/BatchExecuteWithErrorsTest.java

```diff
@@ -142,62 +330,58 @@ private void Repro47239Internal(String mode) throws Exception {
                     batchStmt.addBatch(insertStmt);
                     batchStmt.addBatch(warning);
                     try {
-                        actualUpdateCounts = batchStmt.executeBatch();
+                        actualUpdateCounts = batchStmt.executeLargeBatch();
                         actualExceptionText = "";
                     } catch (BatchUpdateException bue) {
-                        actualUpdateCounts = bue.getUpdateCounts();
+                        actualUpdateCounts = bue.getLargeUpdateCounts();
                         actualExceptionText = bue.getMessage();
-                        if (log.isLoggable(Level.FINE)) {
-                            log.fine("BatchUpdateException occurred. Message:" + actualExceptionText);
-                        }
-                    } finally {
-                        batchStmt.close();
+                        log.fine("BatchUpdateException occurred. Message:" + actualExceptionText);
                     }
                 }
-                if (log.isLoggable(Level.FINE)) {
-                    log.fine("UpdateCounts:");
-                }
-                for (int updateCount : actualUpdateCounts) {
+
+                log.fine("UpdateCounts:");
+                for (long updateCount : actualUpdateCounts) {
                     log.fine("" + updateCount + ",");
                 }
-                log.fine("");
+
```

Comment:
check if FINE 1st the for loop isn't necessary if not

Reply:
I'll remove these logs from Junit tests in next PR - we don't need them